src/Microsoft.Health.Api/Features/Audit/AuditEventTypeMapping.cs

```diff
@@ -31,7 +33,7 @@ namespace Microsoft.Health.Api.Features.Audit
     /// This class builds the mapping ahead of time so that we can lookup the audit event type any time during the pipeline given the controller name and action name.
     /// </para>
     /// </remarks>
-    public class AuditEventTypeMapping : IAuditEventTypeMapping, IStartable
+    public class AuditEventTypeMapping : IAuditEventTypeMapping, IHostedService
```

Comment:
My only thought with using IHostedService is, are we overloading the meaning of this Interface.  The name is throwing me off. 
Looks like eventhough we need same interface, IHostedService is furture used in BackGroundServices.

Reply:
I had a similar thought. Here is a counterpoint: the term `service` is heavily overloaded. One meaning is in the sense of a DI service (`IServiceProvider`). This is a special kind of such a service, one that the runtime explicitly instantiates, starts, and stops. It does not matter whether it continues to do anything in the background in between.

If that is not convincing, we can create an `IHostedService` that instantiates, starts and stops instances of another kind of service (`IStartStoppable` :)) with basically the same signature. But the indirection does not seem very helpful. Also, with IHostedService it is clear when StartAsync and StopAsync will be called (since it is [documented](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.ihostedservice?view=dotnet-plat-ext-3.1)) and we are not inventing our own concept.